src/types.jl

```diff
@@ -35,76 +31,73 @@ struct FontExtent{T}
     scale::Vec{2, T}
 end
 
-hadvance(ext::FontExtent) = ext.advance[1]
-inkwidth(ext::FontExtent) = ext.scale[1]
-inkheight(ext::FontExtent) = ext.scale[2]
-hbearing_ori_to_left(ext::FontExtent) = ext.horizontal_bearing[1]
-hbearing_ori_to_top(ext::FontExtent) = ext.horizontal_bearing[2]
-leftinkbound(ext::FontExtent) = hbearing_ori_to_left(ext)
-rightinkbound(ext::FontExtent) = leftinkbound(ext) + inkwidth(ext)
-bottominkbound(ext::FontExtent) = hbearing_ori_to_top(ext) - inkheight(ext)
-topinkbound(ext::FontExtent) = hbearing_ori_to_top(ext)
+@inline hadvance(ext::FontExtent) = ext.advance[1]
+@inline vadvance(ext::FontExtent) = ext.advance[2]
+@inline inkwidth(ext::FontExtent) = ext.scale[1]
+@inline inkheight(ext::FontExtent) = ext.scale[2]
+@inline hbearing_ori_to_left(ext::FontExtent) = ext.horizontal_bearing[1]
+@inline hbearing_ori_to_top(ext::FontExtent) = ext.horizontal_bearing[2]
+@inline leftinkbound(ext::FontExtent) = hbearing_ori_to_left(ext)
+@inline rightinkbound(ext::FontExtent) = leftinkbound(ext) + inkwidth(ext)
+@inline bottominkbound(ext::FontExtent) = hbearing_ori_to_top(ext) - inkheight(ext)
+@inline topinkbound(ext::FontExtent) = hbearing_ori_to_top(ext)
 
 BroadcastStyle(::Type{<: FontExtent}) = Style{FontExtent}()
 BroadcastStyle(::Style{FontExtent}, x) = Style{FontExtent}()
 BroadcastStyle(x, ::Style{FontExtent}) = Style{FontExtent}()
 
-function broadcasted(op::Function, f::FontExtent, scaling::StaticVector)
-    return FontExtent(
+broadcasted(op::Function, f::FontExtent, scaling::StaticVector) =
+FontExtent(
         op.(f.vertical_bearing, scaling[1]),
         op.(f.horizontal_bearing, scaling[2]),
         op.(f.advance, scaling),
         op.(f.scale, scaling),
     )
-end
 
-function broadcasted(op::Function, f::FontExtent)
-    return FontExtent(
+broadcasted(op::Function, f::FontExtent) =
+    FontExtent(
         op.(f.vertical_bearing),
         op.(f.horizontal_bearing),
         op.(f.advance),
         op.(f.scale),
     )
-end
 
-function broadcasted(op::Function, ::Type{T}, f::FontExtent) where T
```

Comment:
So these function + return statement are actually part of following  the [YASGuide}(https://github.com/jrevels/YASGuide)...
To be fair I don't really follow it that strictly as well, but if it's not too much trouble, it would be nice to not mingle in style changes in this PR.

Reply:
Sorry about that. I've reverted the changes to using `function ... end` when the statement spans over more than one line as advised, and added explicit `return`s.